Gemfile

```diff
@@ -149,7 +149,7 @@ group :test do
   gem 'capybara', "~> 2.2.1"
   gem 'cucumber-rails', '~> 1.4.0', :require => false
   gem 'cucumber'
-  gem 'selenium-webdriver', '~> 2.45.0'
```

Comment:
Sounds OK to me. 
(Current version is 2.49 i.e. we have not updated this very often.)